pkg/curatedpackages/oras/pusher.go

```diff
@@ -0,0 +1,48 @@
+package oras
+
+import (
+	"context"
+	"fmt"
+
+	"oras.land/oras-go/pkg/content"
+	"oras.land/oras-go/pkg/oras"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+)
+
+type Pusher struct {
+	registry *content.Registry
+	store    *content.Memory
+}
+
+func NewPusher(registry *content.Registry, store *content.Memory) *Pusher {
```

Comment:
I noticed we're injecting structs instead of interfaces. In the case of the `content.Memory` type it doesn't have any options and we inject a raw `content.Memory` from `import_images.go` so could this just be constructed in-line here? 

The registry does take options but we don't configure them in any capacity. Do we reasonably expect to need to customize the registry at all?